cmd/sdn-cni-plugin/openshift-sdn_linux.go

```diff
@@ -126,9 +126,11 @@ var iptablesCommands = [][]string{
 	{"-A", "FORWARD", "-p", "tcp", "-m", "tcp", "--dport", "22623", "-j", "REJECT"},
 	{"-A", "FORWARD", "-p", "tcp", "-m", "tcp", "--dport", "22624", "-j", "REJECT"},
 
-	// Block cloud metadata IP
-	{"-A", "OUTPUT", "-d", "169.254.169.254", "-j", "REJECT"},
-	{"-A", "FORWARD", "-d", "169.254.169.254", "-j", "REJECT"},
+	// Block cloud metadata IP on ports 80 & 443 for TCP & UDP
+	{"-A", "OUTPUT", "-p", "tcp", "-m", "tcp", "-d", "169.254.169.254", "--dport", "80", "-j", "REJECT"},
+	{"-A", "OUTPUT", "-p", "udp", "-m", "udp", "-d", "169.254.169.254", "--dport", "80", "-j", "REJECT"},
+	{"-A", "FORWARD", "-p", "tcp", "-m", "tcp", "-d", "169.254.169.254", "--dport", "443", "-j", "REJECT"},
+	{"-A", "FORWARD", "-p", "udp", "-m", "udp", "-d", "169.254.169.254", "--dport", "443", "-j", "REJECT"},
```

Comment:
This got messed up in one of the rewrites... now it's blocking port 80 in `OUTPUT` and port 443 in `FORWARD`.

Also, we don't need to block UDP, just TCP. Also, you can simplify using `multiport`. So (I think, not tested):

    {"-A", "OUTPUT", "-d", "169.254.169.254", "-p", "tcp", "-m", "multiport", "--destination-ports", "80,443", "-j", "REJECT"},
    {"-A", "FORWARD", "-d", "169.254.169.254", "-p", "tcp", "-m", "multiport", "--destination-ports", "80,443", "-j", "REJECT"},

but also, @squeed, what's the verdict on 8080?

Reply:
Waiting for the west coast to have coffee, then we'll see.